pymc4/random_variables/continuous.py

```diff
@@ -52,41 +1294,22 @@ def _base_dist(self, *args, **kwargs):
         """
         return tfd.TransformedDistribution(
             distribution=tfd.Uniform(low=0.0, high=1.0),
-            bijector=tfp.bijectors.Invert(tfp.bijectors.Weibull(*args, **kwargs)),
+            bijector=tfp.bijectors.Invert(
+                tfp.bijectors.Weibull(scale=beta, concentration=alpha, *args, **kwargs)
+            ),
             name="Weibull",
         )
 
 
 # Random variables that tfp supports as distributions. We wrap these
 # distributions as random variables. Names must match tfp.distributions names
 # exactly.
-tfp_supported = [
-    "Beta",
-    "Cauchy",
-    "Chi2",
-    "Exponential",
-    "Gamma",
-    "Gumbel",
-    "HalfCauchy",
-    "HalfNormal",
-    "InverseGamma",
-    "InverseGaussian",
-    "Kumaraswamy",
-    "Laplace",
-    "LogNormal",
-    "Logistic",
-    "Normal",
-    "Pareto",
-    "StudentT",
-    "Triangular",
-    "Uniform",
-    "VonMises",
-]
+# tfp_supported = ["InverseGaussian"]  # not present in PyMC3. We can choose how to parameterize
 
 # Programmatically wrap tfp.distribtions into pm.RandomVariables
-for dist_name in tfp_supported:
-    setattr(
-        sys.modules[__name__],
-        dist_name,
-        type(dist_name, (RandomVariable,), {"_base_dist": getattr(tfd, dist_name)}),
-    )
+# for dist_name in tfp_supported:
```

Comment:
These should just be deleted then rather than commented.

Reply:
Thanks @twiecki! Resolved.